client/src/components/DataFiles/DataFilesListing/DataFilesListingCells.js

```diff
@@ -38,7 +40,14 @@ export const CheckboxCell = React.memo(({ index }) => {
   const selected = useSelector(state =>
     state.files.selected.FilesListing.includes(index)
   );
-  return <Checkbox isChecked={selected} />;
+  return (
+    <Checkbox
+      isChecked={selected}
+      id={`FileCheckbox_${index}`}
+      role="checkbox"
+      aria-label={`select file ${index}`}
```

Comment:
Minor question. How much work is it to get the filename in here?

_The user will not know what file/folder they are selecting (unless they tab over to file then tab back, which is like what we do with out eyes anyway). [Thoughts in internal chat.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1639684984151100?thread_ts=1639684563.150700&cid=CQUA4D5KJ)_

Reply:
Ah, great point, I wondered about the semantics. I struggled to think of a good term here and settled on the generic "select file," so I should've asked about it first. I'll be sure to change this. 

I'll look into using name vs. index too. I initially did try to use the name and got an error (which I believe is because the name isn't being passed into the `CheckboxCell`?), but I would think that's a simple fix. Will do that too!

Reply:
@tacc-wbomar I've updated the code. Now it should specify whether it's a file or folder and it should also use the appropriate name. For example:
- checked, select folder .APPDATA, checkbox
- unchecked, select file coe_332.json, checkbox

https://user-images.githubusercontent.com/51714485/146457303-15c9935a-23b8-48e3-9d70-b473d83006be.mov

Reply:
I think there is no good one term. I know of "entry", but that's a bit obtuse. So, I agree with "file or folder".

I am glad it's possible to get the file/folder name in. I think there may a simpler way to get the name.

Would [`row.original.name`](https://github.com/TACC/Core-Portal/blob/7179a41/client/src/components/DataFiles/DataFilesListing/DataFilesListing.js#L52) give the correct file/folder name also? I think [that `name` is rendered to the UI](https://github.com/TACC/Core-Portal/blob/7179a41/client/src/components/DataFiles/DataFilesListing/DataFilesListingCells.js#L57).

Reply:
Ah, I see you tried `name`. Odd, I'll try also, and review the error.

Reply:
I was able to get the name to show up. I initially hit an error that I knew meant I needed to validate the prop type.

To simplify the new code, I suggest implementing the `name` property and fixing the error. My screenshots may help.

<details>
<summary>Screenshots</summary>

| pass prop | support prop | render prop |
| - | - | - |
| ![FP-1385 Name - DataFilesListing](https://user-images.githubusercontent.com/62723358/146582554-701d3324-aa95-41ea-b1e5-6bcd47ce7e7e.png) | ![FP-1385 Name - DataFilesListingCells](https://user-images.githubusercontent.com/62723358/146582556-bc80c96e-bf15-4914-843a-b007b77dedae.png) | ![FP-1385 Name - Markup](https://user-images.githubusercontent.com/62723358/146582557-d174f7db-6c18-49f8-b280-f8d04ca0f1f9.png) |

</details>

If this works, then you can re-review the final diff[^1] to make sure no unused new code still remains.

[^1]: By pushing changes to GitHub, command line, or via certain code editor features.

Reply:
@tacc-wbomar Before reading this last comment with the code, I'd pushed a commit that made the same changes you have, so now it's right on track! Thanks for the suggestion, I'm glad it's simpler now :)

I think what happened was exactly what you said. I forgot to validate the prop type and thought it must've been a problem with passing `name`. I updated it as suggested above, removed the `href` references, and confirmed it still works properly.

Reply:
Yep also can confirm that it works!